Formula/a/atari800.rb

```diff
@@ -21,18 +21,27 @@ class Atari800 < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "656005889be7d36009a7b927d4b7437f38b988cc74063b5e3cefe3406bbffd77"
   end
 
+  head do
+    url "https://github.com/atari800/atari800.git", branch: "master"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
   depends_on "libpng"
   depends_on "sdl12-compat"
 
   def install
+    system "./autogen.sh" if build.head?
     system "./configure", "--disable-sdltest",
                           "--disable-riodevice",
                           *std_configure_args.reject { |s| s["--disable-debug"] }
     system "make", "install"
   end
 
   test do
-    assert_equal "Atari 800 Emulator, Version #{version}",
-                 shell_output("#{bin}/atari800 -v", 3).strip
+    assert_match(/Atari 800 Emulator, Version \d.\d.\d/,
+                 shell_output("#{bin}/atari800 -v", 3).strip)
```

Comment:
no need update test (we dont run test for head build)

Reply:
updated, thanks for letting me know